src/esm_parser/yaml_to_dict.py

```diff
@@ -169,39 +173,51 @@ def yaml_file_to_dict(filepath):
     FileNotFoundError
         Raised when the YAML file cannot be found and all extensions have been tried.
     """
-    loader = create_env_loader()
+    #loader = create_env_loader()
     for extension in YAML_AUTO_EXTENSIONS:
         try:
             with open(filepath + extension) as yaml_file:
                 # Check for duplicates
                 check_duplicates(yaml_file)
                 # Back to the beginning of the file
                 yaml_file.seek(0, 0)
+                yaml_load = {}
                 # Actually load the file
-                yaml_load = yaml.load(yaml_file, Loader=loader)  # yaml.FullLoader)
+#                yaml_load = yaml.load(yaml_file, Loader=loader)  # yaml.FullLoader)
+
+                esm_tools_loader = parse_it.EsmToolsLoader()
```

Comment:
I think the `EsmToolsLoader` class belongs in this file

Reply:
Yes. I have this fixed in my local branch already. But I will move EsmToolsLoader etc. to yaml_to_dict.py anyway.